src/Language/PureScript/DCE/CoreFn.hs

```diff
@@ -111,24 +111,39 @@ runDeadCodeElimination entryPoints modules = uncurry runModuleDeadCodeEliminatio
 
       -- | Find dependencies of an expression.
       deps :: Expr Ann -> [Key]
-      deps = go
+      deps = onExpr'
         where
-          (_, go, _, _) = everythingOnValues (++)
-            (const [])
-            onExpr
-            onBinder
-            (const [])
-
           -- | Build graph from qualified identifiers.
           onExpr :: Expr Ann -> [Key]
           onExpr (Var _ i) = [i | isQualified i]
           onExpr _ = []
 
-          onBinder :: Binder Ann -> [Key]
-          onBinder (ConstructorBinder _ _ c _) =
-            [fmap (Ident . runProperName) c]
+          onExpr' :: Expr Ann -> [Key]
```

Comment:
Could you rename this function, it could be `go` or something more descriptive like `traverseExpr`